### DISCUSSION TITLE
Animation not rendering

### DISCUSSION THREAD

**Discussion by PampoenKoekie** [shteeve3d/blender-wiggle-2#48] [Q&A]
First off, amazing work on this great addon! I do have some troubles which I do not know how to resolve. I have a simple carrot character rigged, everything works as intended. When I press play and then G to grab the bones, it dynamicly reacts and all is great. I record my movemnt and if I play it back it all works. But the problem is, once I start to render the animation, no dynamics is present. Just some stiff armature. I did bake out the animation for the bones but then all other animation is gone. Is there any advice, am I doing something wrong maybe? I need the bones to react to my movement and then just render everything out.

**Comment by Dimko1990** [comment]
Hi)
check this option is on?
![image](https://github.com/shteeve3d/blender-wiggle-2/assets/37138460/eed0fef6-11db-4a56-a450-9500041f1422)

**Comment by PampoenKoekie** [comment]
Worked wonderfully, thank you for your time and assistance! Greatly appreciated! All the best to you
